recipe/meta.yaml

```diff
@@ -1,6 +1,9 @@
 {% set name = "jupyterlab-nvdashboard" %}
 {% set version = "0.12.0" %}
 
+{% set python_min = "3.8" %}
```

Comment:
If we are ok using Python 3.9 (oldest supported version), that may make this easier to maintain (as we can drop this)

Reply:
I'm fine with that

Reply:
```suggestion
```